pkg/windows/windows.go

```diff
@@ -256,23 +248,42 @@ type windows struct {
 	log      logr.Logger
 	// defaultShellPowerShell indicates if the default SSH shell is PowerShell
 	defaultShellPowerShell bool
+	// filesToTransfer is the list of files needed for the windows VM
+	filesToTransfer map[*payload.FileInfo]string
 }
 
 // New returns a new Windows instance constructed from the given WindowsVM
-func New(clusterDNS string, instanceInfo *instance.Info, signer ssh.Signer) (Windows, error) {
+func New(clusterDNS string, instanceInfo *instance.Info, signer ssh.Signer, platform *config.PlatformType) (Windows, error) {
 	log := ctrl.Log.WithName(fmt.Sprintf("wc %s", instanceInfo.Address))
 	log.V(1).Info("initializing SSH connection")
 	conn, err := newSshConnectivity(instanceInfo.Username, instanceInfo.Address, signer, log)
 	if err != nil {
 		return nil, fmt.Errorf("unable to setup VM %s sshConnectivity: %w", instanceInfo.Address, err)
 	}
 
+	files := filesToTransfer
```

Comment:
With your change we shouldn't need the global `filesToTransfer` map anymore. Why are you retaining it?

Reply:
Is there a place where the list of files to transfer gets overridden? I thought that was why it was written that way in the first place, and I don't want to create a regression, but if that's not the case, getting rid of it would be good!

Reply:
>Is there a place where the list of files to transfer gets overridden?

Please look for all instances of `filesToTransfer` and figure that out. If there is a regression you won't be able to bring up a node which should be part of your dev testing.